src/main/java/faang/school/godbless/streamapi_1/StreamLib.java

```diff
@@ -0,0 +1,44 @@
+package faang.school.godbless.streamapi_1;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.function.Predicate;
+import java.util.stream.Stream;
+
+public class StreamLib {
+    public static List<Integer> evenList(List<Integer> list) {
```

Comment:
You don't need to implement this kind of functionality. The requirement is to create a method which returns sum of all even numbers

Reply:
fixed